ARGO/src/InputSystem.cpp

```diff
@@ -17,5 +21,64 @@ void InputSystem::update(Entity& t_entity)
 	{
 		InputComponent* inputComp = static_cast<InputComponent*>(t_entity.getAllComps().at(COMPONENT_ID::INPUT_ID));
 		inputComp->update();
+		handleInputs(inputComp);
+		handleCommands(t_eventManager, inputComp, t_entity);
+	}
+}
+
+void InputSystem::handleInputs(InputComponent* t_inputComponent)
+{
+	Controller controller = t_inputComponent->getController();
```

Comment:
If getController returns the same object every time would it be more efficient to have the input system hold a reference instead of creating a new controller object every time this is called?

Reply:
Probably not worth the effort though :P